docs/src/content/docs/ops/config/authentication/oidc.adoc

```diff
@@ -0,0 +1,114 @@
+---
+title: OpenID Connect Authentication
+---
+
+XTDB integrates with OpenID Connect identity providers (Keycloak, Auth0, Azure AD) for enterprise authentication. OIDC authentication supports multiple OAuth 2.0 flows for different client types.
+
+== Identity Provider Requirements
+
+**Required Capabilities**::
+- OpenID Connect Discovery (/.well-known/openid_configuration)
```

Comment:
adoc nit: consistency about your bullet points

`*` for top-level, `**` for second